### ISSUE TITLE
Ratlantis Portal does not work on Server

### ISSUE THREAD

**Issue by Michi2670** [Alex-the-666/Rats#459]
Im running a server with the MC version 1.16.3, Rats version 7.0.1 and Ratlantis version 1.0.0 and i cant enter Ratlantis via the Portal. Do i have to enable it somewhere in the config?
When i download the world and enter it in singleplayer everything works fine

**Comment by DukeTwoCan** [comment]
I am also having this issue. I tried to generate the dimension in singleplayer and upload the dimensions files to my server and the server still cannot find it. I have attempted to upload the dimension files under /dimensions/rats/ratlantis and I've even tried uploading both /rats/ratlantis/ and /ratlantis/ to just under my worlds main folder where DIM-1 and DIM0 are with no success in getting the game to register the dimension even exists. Meaning that the /execute in rats:ratlantis commands return an error that the dimension could not be found. Same version as original poster.
